novawallet/Modules/Staking/Services/RewardCalculatorService/Parachain/BaseParaStakingRewardCalculatoService+Fetch.swift

```diff
@@ -1,37 +1,52 @@
 import Foundation
+import SubstrateSdk
+import BigInt
+import Operation_iOS
 
 extension BaseParaStakingRewardCalculatoService {
-    func updateTotalStaked() throws {
-        totalStakedService?.throttle()
-        totalStakedService = nil
+    func updateTotalStaked() {
```

Comment:
This changes shouldn't be here. Please, make sure you have merged develop to the base branch and check that we don't have this changed.

Reply:
now up to develop